plugins/plugin-js/build.gradle.kts

```diff
@@ -15,13 +15,14 @@ repositories {
 
 dependencies {
     api(kotlin("stdlib"))
-    //implementation(kotlin("stdlib-jdk8"))
 
     implementation(kotlin("stdlib-js"))
     implementation(gradleApi())
     implementation(gradleKotlinDsl())
     implementation(kotlin("gradle-plugin-api"))
 
+    runtimeOnly("org.jetbrains.kotlin.js:org.jetbrains.kotlin.js.gradle.plugin:1.4.0")
```

Comment:
why runtimeOnly? why not applu the plugin in plugin code? See examples in graal version